build/resources/package/chocolatey/livestreamer-twitch-gui.nuspec.tpl

```diff
@@ -26,14 +26,14 @@ Example: `choco uninstall livestreamer-twitch-gui -y -params '"/Purge"'`
   [Chromium]: https://www.chromium.org/ "Chromium"
     </description>
     <projectUrl><%= homepage %></projectUrl>
-    <packageSourceUrl>https://github.com/bastimeyer/livestreamer-twitch-gui</packageSourceUrl>
-    <tags>livestreamer twitch gui streaming</tags>
+    <packageSourceUrl>https://github.com/streamlink/streamlink-twitch-gui</packageSourceUrl>
+    <tags>streamlink twitch gui livestreamer streaming</tags>
     <copyright>Sebastian Meyer</copyright>
-    <licenseUrl>https://github.com/bastimeyer/livestreamer-twitch-gui/blob/master/LICENSE</licenseUrl>
-    <docsUrl>https://github.com/bastimeyer/livestreamer-twitch-gui/wiki</docsUrl>
-    <bugTrackerUrl>https://github.com/bastimeyer/livestreamer-twitch-gui/issues</bugTrackerUrl>
+    <licenseUrl>https://github.com/streamlink/streamlink-twitch-gui/blob/master/LICENSE</licenseUrl>
+    <docsUrl>https://github.com/streamlink/streamlink-twitch-gui/wiki</docsUrl>
+    <bugTrackerUrl>https://github.com/streamlink/streamlink-twitch-gui/issues</bugTrackerUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <iconUrl>https://cdn.rawgit.com/bastimeyer/livestreamer-twitch-gui/master/src/img/icon-256.png</iconUrl>
+    <iconUrl>https://cdn.rawgit.com/streamlink/streamlink-twitch-gui/master/src/img/icon-256.png</iconUrl>
     <dependencies>
       <dependency id="livestreamer" version="1.12.2"/>
```

Comment:
Will the application still work with livestreamer? If not, we should remove this dependency. I will make a pull request with the Streamlink dependency once I have created the package and validated that everything is working properly.

Notable that this will leave the package in a non-function state (at least on its own), but I don't want to hold up the PR. 😄